### PR TITLE
<nixpkgs> anti pattern

### LINKED ISSUE #19: Anti pattern: relying on <nixpkgs>

### PR DESCRIPTION
**Pull request by domenkozar**
Fixes #19

### CODE REVIEW

**Comment by domenkozar** [comment]
> It might be a good idea to frame this more as a discussion of the feature, highlighting the tradeoff it offers so the reader can make an informed decision. Calling the nix path "problematic" and linking to "how to do better" could leave readers feeling unnecessarily bad for their use nixpath.

That's my intention, I think `<nixpkgs>` usage should be dropped all together. 

It's not quite possible to avoid `$NIX_PATH` with NixOS, but you can avoid `<nixpkgs>`.

I argue it's not unnecessary to avoid it based on my experience, although I probably should elaborate further.

> Ah, this might also be a good place to mention that flakes will change how this works in the future.

I don't want to put flakes in just yet, that's more like a power user feature at the moment. That will hopefully change quite soon.

File: source/anti-patterns/language.rst [hunk 1]
```diff
@@ -77,4 +77,22 @@ The better way is to use a variable:
...
+
+``<...>`` search path
+---------------------
+
+Usually ``<nixpkgs>``, ``<..>`` is syntax for importing Nix expressions specified by shell environment variable ``$NIX_PATH``.
+
+Two developers on different machines are likely to have `<nixpkgs>` point to different revisions,
+which will lead to getting different results.
+
+It's possible to specify exact nixpkgs commit via ``$NIX_PATH``, but that's still problematic unless:
```
[the remark this answers is not in the document]
Comment (domenkozar) [reply source/anti-patterns/language.rst]: linked.

**Comment by domenkozar** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by domenkozar** [comment]
recheckcla

**Comment by Valodim** [comment]
fair enough :+1:

**Pull request merged**